<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPassword</class>
 <widget class="QDialog" name="DialogPassword">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>305</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu Mono</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Enter and select connection details</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="previousConnectionsLabel">
         <property name="text">
          <string>Previous connections:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="previousConnectionsComboBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="rememberPasswordsLabel">
         <property name="text">
          <string>Remember passwords:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="rememberPasswordsCheckBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="iPAddressLabel">
         <property name="text">
          <string>IP address:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="iPAddressLineEdit">
         <property name="maxLength">
          <number>256</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="usernameLabel">
         <property name="text">
          <string>Username:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="usernameLineEdit"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="passwordLabel">
         <property name="text">
          <string>Password: </string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="passwordLineEdit">
         <property name="maxLength">
          <number>256</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="sudoPasswordLabel">
         <property name="text">
          <string>Sudo password *:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="sudoPasswordLineEdit">
         <property name="maxLength">
          <number>256</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="useKeyFileLabel">
         <property name="text">
          <string>Use key file:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="useKeyFileCheckBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_addNew">
         <property name="text">
          <string>Add New</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_remove">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_cancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_ok">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget_messages">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>sudoPasswordLineEdit</tabstop>
  <tabstop>previousConnectionsComboBox</tabstop>
  <tabstop>rememberPasswordsCheckBox</tabstop>
  <tabstop>iPAddressLineEdit</tabstop>
  <tabstop>usernameLineEdit</tabstop>
  <tabstop>useKeyFileCheckBox</tabstop>
  <tabstop>listWidget_messages</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
